astropy/convolution/convolve.py

```diff
@@ -348,7 +348,10 @@ def convolve(
                 raise ValueError(
                     "The kernel can't be normalized, because "
                     "its sum is close to zero. The sum of the "
-                    f"given kernel is < {1.0 / MAX_NORMALIZATION}"
+                    f"given kernel is < {1.0 / MAX_NORMALIZATION}. "
```

Comment:
Do you want to set a float precision here for nicer message or it doesn't matter?

Reply:
done